api/xds_protocol.rst

```diff
@@ -766,6 +755,7 @@ An example minimal ``bootstrap.yaml`` fragment for ADS configuration is:
         # may be inadequate if there is a TCP proxy between Envoy and the management server.
         # HTTP/2 keepalive is slightly more expensive, but may detect issues through more types
         # of intermediate proxies.
+        # WRONG I THINK
```

Comment:
as per comment

Reply:
as this is a bootstrap possibly best to move to a  literalinclude to ensure its tested

Reply:
moved to a literalinclude...